File: Python/16-nth-prime.py
def is_prime(num):
    if num <= 1:
        return False

    for i in range(2, num/2+1):
        if (num % i == 0):
            return False

    return True

def nth_prime(num):
    count = 0

    i = 2
    while True:
        if is_prime(i):
            count += 1
        if count == num:
            return i
        i += 1

    return

print(nth_prime(5))

File: Javascript/16-nth-prime.js
function isPrime(num) {

	if (num <= 1) {
		return false;
	}

	for (var i = 2; i < num; i++) {
		if (num % i == 0) {
			return false;
		}
	}

	return true;
}

function nthPrime(n) {
	var primes = [];
	var i = 2;

	while (true) {
		if (isPrime(i)) {
			primes.push(i);
		}

		if (primes.length == n) {
			return primes[n-1];
		}

		i++;
	}

	return null;

}

console.log(nthPrime(1000));
